apps/rush-lib/src/cli/logic/pnpm/PnpmShrinkwrapFile.ts

```diff
@@ -95,6 +95,11 @@ export function extractVersionFromPnpmVersionSpecifier(version: string): string
     if (versionParts.length === 5 && isScoped) {
       extractedVersion = versionParts[3]; // e.g. "3.1.1"
     }
+
+    // e.g. "path.pkgs.visualstudio.com/@scope/depame/1.4.0"
+    if (!extractedVersion && versionParts[versionParts.length - 1].match(/^(\d*)\.(\d*)\.(\d*)/)) {
```

Comment:
>\d* [](start = 83, length = 3)

should this be "\d+"?

Reply:
You are correct here. I have moved away from the regular expression now. it. III

---
In reply to: [182982855](https://github.com/Microsoft/web-build-tools/pull/615#discussion_r182982855) [](ancestors = 182982855)